ublksrv_tgt.cpp

```diff
@@ -463,6 +466,7 @@ static void ublksrv_tgt_set_params(struct ublksrv_ctrl_dev *cdev,
 				dev_id, ret);
 	}
 }
+#pragma GCC diagnostic pop
```

Comment:
I guess ublksrv_tgt_set_params needn't to be covered by the above
macro. Otherwise, this patch looks fine.

BTW,  it is really good to enable more compiler warning, thanks for
doing that!

Reply:
You're absolutely right. I've pushed an updated commit with the pragma shifted up.